README.md

```diff
@@ -227,6 +227,27 @@ list.addEventListener('rangechange', (event) => {
 });
 ```
 
+### Scrolling
+
+`<virtual-list>` needs to be sized in order to determine how many items should be rendered. Its default size is:

```

Comment:
I think showing the default size with CSS is not that helpful. I'd instead just say "300px × 150px, similar to [CSS inline replaced elements](https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-width) like images and iframes"